### PR TITLE
Updated security policy

### PR DESCRIPTION
**Pull request by eyurtsev**
Updated security policy

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -1,6 +1,61 @@
 # Security Policy
 
-## Reporting a Vulnerability
+## Reporting OSS Vulnerabilities
 
-Please report security vulnerabilities by email to `security@langchain.dev`.
-This email is an alias to a subset of our maintainers, and will ensure the issue is promptly triaged and acted upon as needed.
+LangChain is partnered with [huntr.dev](https://huntr.com/) to provide
```
Comment (psmoros) [inline SECURITY.md:5]: Instead of huntr.dev you can say huntr / huntr.com or huntr by Protect AI up to you :)

**Review by psmoros — APPROVED** [review]
Looks great looking forward to a fruitful collab 🥳

File: SECURITY.md [hunk 2]
```diff
@@ -1,6 +1,61 @@
...
+
+Please report security vulnerabilities associated with the LangChain 
+open source projects by visiting the following link:
+
+[https://huntr.com/bounties/disclose/](https://huntr.com/bounties/disclose/?target=https%3A%2F%2Fgithub.com%2Flangchain-ai%2Flangchain&validSearch=true)
+
+Before reporting a vulnerability, please review:
+
+1) In-Scope Targets and Out-of-Scope Targets below.
+2) The LangChain repository structure as [langchain-ai/langchain](https://python.langchain.com/docs/contributing/repo_structure).
```
Comment (baskaryan) [inline SECURITY.md:16]: awkward, maybe "The [langchain-ai/langchain](https://python.langchain.com/docs/contributing/repo_structure) monorepo structure"

**Review by baskaryan — APPROVED** [review]

File: SECURITY.md (hunk 2 above)
Comment (eyurtsev) [inline SECURITY.md:16]:
```suggestion
2) The [langchain-ai/langchain](https://python.langchain.com/docs/contributing/repo_structure) monorepo structure.
```

**Pull request merged**